### DISCUSSION TITLE
How is SSH supposed to work?

### DISCUSSION THREAD

**Discussion by canadiannomad** [vhqtvn/VHSCode-Android#207] [Q&A] [answered]
I'm trying to figure out the remote code-server feature..
I try opening a code-server on localhost in official termux, and all I get is a blank screen with some configs at the bottom.
So I tried configuring ssh inside vheditor's terminal..
Then I added my host from my .ssh/config into the 'ssh aliases' file.. It just says Starting.. then Finished.
I'm currently at a loss as to how I should configure SSH, keypairs, remote code-server, etc.
Is there any documentation?  Thanks!

**Comment by vhqtvn** [comment, marked as answer]
I've written a quick wiki here: https://github.com/vhqtvn/VHEditor-Android/wiki/SSH-features
Please contribute to the wiki if something is unclear.

**Comment by emmzyrayz** [comment]
How do I generate ssh please someone put me through
